.storybook/main.js

```diff
@@ -151,10 +152,10 @@ module.exports = {
 						test: /\.js$/,
 						enforce: "pre",
 						use: ["source-map-loader"],
-					} /* Raw SVG loader */,
+					} /* Raw loader */,
 					{
-						test: /\.svg$/i,
-						loader: "raw-loader",
+						resourceQuery: /raw/,
```

Comment:
Loading raw assets using the asset/source syntax is the preferred way in the latest webpack update - they recommend against using raw-loader.